tron/core/actionrun.py

```diff
@@ -1314,6 +1319,19 @@ def kill(self, final: bool = True) -> Optional[str]:
 
         return "\n".join(msgs)
 
+    def _exit_unsuccessful(
+        self, exit_status=None, retry_original_command=True, non_retryable_exit_codes=()
+    ) -> Optional[Union[bool, ActionCommand]]:
+
+        k8s_cluster = KubernetesClusterRepository.get_cluster()
+        non_retryable_exit_codes = () if not k8s_cluster else k8s_cluster.non_retryable_exit_codes
```

Comment:
Same note here about the tuple

```suggestion
        non_retryable_exit_codes = [] if not k8s_cluster else k8s_cluster.non_retryable_exit_codes
```